features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -83,6 +83,12 @@
 		<feature dependency="true">openhab.tp-jmdns</feature>
 	</feature>
 
+	<feature name="openhab-core-config-discovery-addon-process" version="${project.version}">
```

Comment:
If you add this feature to `openhab-runtime-base` you will not need to add this finder to the constants in AddonFinderConstants.java. The feature will then automatically get installed and not through the service. The only reason to do it through the service is to allow enabling/disabling.

Reply:
I think someone did already ask to have enabling/disabling. ??

Reply:
For me, that's conditional. If retrieving the processes is too high a load, we should. Otherwise I don't see the point. Right now, the implementation does not cache anything and does a call to retrieve the processes any time the suggestions are retrieved. If that is quick, no need to enable/disable a little piece of code. There is nothing running in the background in separate threads either.
@kaikreuzer Did not react on my feedback on it. If he feels strongly, then, of course it needs to be there, but it should also be added to the configuration parameters.

Reply:
@mherwege I am OK with that.